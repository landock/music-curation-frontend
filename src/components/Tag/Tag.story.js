import React from 'react'
import { storiesOf } from '@storybook/react'
import { action } from '@storybook/addon-actions'
import { linkTo } from '@storybook/addon-links'
import Tag from '.'

storiesOf('Tag', module).add('default', () => <Tag data={{id: 1, title: 'whatever'}} />)
