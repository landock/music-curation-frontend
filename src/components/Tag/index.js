import React from 'react';
import { Label } from 'semantic-ui-react';

function Tag({ data }) {
  return (
    <Label style={{ marginBottom: '1%' }} color="red">
      {data.title}
    </Label>
  );
}

export default Tag;
