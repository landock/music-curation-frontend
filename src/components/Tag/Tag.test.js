import React from 'react';
import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer';
import Tag from '.';

it('Tag: default', () => {
  // const component = renderer.create(<Tag data={{ id: 1, title: 'test' }} />);
  // const tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});
