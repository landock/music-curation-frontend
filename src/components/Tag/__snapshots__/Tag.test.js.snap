// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag: default 1`] = `
<span
  style={
    Object {
      "background": "red",
      "color": "white",
      "display": "inline-block",
      "margin": "5px",
      "padding": "5px",
    }
  }
>
  test
</span>
`;
